api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -211,6 +211,12 @@ message JwtProvider {
   // Specify the clock skew in seconds when verifying JWT time constraint,
   // such as `exp`, and `nbf`. If not specified, default is 60 seconds.
   uint32 clock_skew_seconds = 10;
+
+  // This field is effective only when `forward_payload_header` is specified.
+  // If true, complete the padding in the forwarded JWT payload binary if needed.
+  // By default, it is false and the JWT payload binary will be directly forwarded
+  // without adding padding.
```

Comment:
The comment as:
// This field only applies to the field :ref:`forward_payload_header<envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.JwtProvider....>.
// When it is specified,  the base64 encoded payload will be added to the header.  Normally JWT based64 encode doesn't add padding. If this field is true,  the padding will be added to the header.

Reply:
Done